codacy/values.yaml

```diff
@@ -27,8 +27,8 @@ global:
     # if you set this provider configuration, you must save it on the admin/integrations ui.
     # enabled: "false"
     # app:
-    #  id: "1234"
     #  name: "codacy"
+    #  id: "1234"
```

Comment:
@mrfyda I changed the order of this setting because it makes more sense to have the name at the top, just like on the GitHub interface for creating GitHub Apps.